### PR TITLE
#5363-Define-QA-Automation-Process

### PR DESCRIPTION
**Pull request by BojteTamas**
First draft of the QA development process

### CODE REVIEW

**Review by tamasvlad — COMMENTED** [review]
Depending the size of the team we may want to consider having at least 2 approvals at code review before we merge.
Also I propose to discuss one more step at the end: "## 6. Delete the working branch after the merge is completed."

**Comment by BojteTamas** [comment]
> Depending the size of the team we may want to consider having at least 2 approvals at code review before we merge.
> Also I propose to discuss one more step at the end: "## 6. Delete the working branch after the merge is completed."

Good ideas @tamasvlad, the second part is done as you can see at the final of this PR :https://github.com/hzi-braunschweig/SORMAS-Project/pull/5345

**Review by tamasvlad — APPROVED** [review]

**Pull request merged**
